### ISSUE TITLE
[IMPROVEMENT] Support install Longhorn prerequisite for Photon OS

### ISSUE THREAD

**Issue by PhanLe1010** [longhorn/longhorn#8390]
## Is your improvement request related to a feature? Please describe (👍 if you like this request)

Photon OS use `tdnf` package manager which doesn't appear in our prerequisite yaml. We need to double check if we need to modify the these yamls to support Photon OS

Ref: https://github.com/longhorn/longhorn/tree/master/deploy/prerequisite

**Comment by c3y1huang** [comment]
We should also consider integrating this into CLI.

**Comment by PhanLe1010** [comment]
> We should also consider integrating this into CLI.

Good idea! @c3y1huang Could you help with the CLI part? I will take care of the yaml part here

**Comment by c3y1huang** [comment]
> Good idea! @c3y1huang Could you help with the CLI part? I will take care of the yaml part here

Sure. Let's handle the CLI part at https://github.com/longhorn/longhorn/issues/8391.
